src/test/java/com/gachigang/ontherun/controller/DepartmentControllerTest.java

```diff
@@ -20,12 +21,21 @@ public class DepartmentControllerTest {
     private DepartmentController departmentController;
 
    @Test
-    void getDepartmentByUserId() {
+    void testGetDepartmentByUserId() {
         User user = User.builder()
                         .id(1L)
                         .lastname("test")
                         .build();
         departmentController.getDepartmentByUserId(user);
         verify(departmentService, times(1)).getDepartmentByUserId(user.getId());
     }
+
```

Comment:
The test not bad but we need to test all variants like positive and negative

Reply:
Hope, i've done it